src/content/ui/design/cupertino/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Cupertino design for Flutter
+description: Learn about Cupertino design for Flutter.
+---
+
+The Flutter Cupertino library is a collection
+of widgets that implement Apple's iOS design language
+for Flutter apps. The widgets have a similar look
+and feel to iOS, including rounded corners, gradients,
+and minimalistic design.
+They also include iOS interactions and animations. 
+
+## More information {:.no_toc}
+
+To learn more about Cuperino design and Flutter,
+check out:
+
+* The (mostly visual) [Cupertino widget catalog][]
+  on doc.flutter.dev
+* The [Cupertino library][] page in the API docs
+
+PENDING:
+What other resources can we link to?
```

Comment:
There's also a bajillion samples here: https://github.com/flutter/flutter/tree/master/examples/api/lib/cupertino

Instructions to run these samples are here: https://github.com/flutter/flutter/tree/master/examples/api#api-example-code